tests/integration/ceph_upgrade_test.go

```diff
@@ -138,8 +138,8 @@ func (s *UpgradeSuite) testUpgrade(useHelm bool, initialCephVersion v1.CephVersi
 	err := s.installer.WaitForToolbox(s.namespace)
 	assert.NoError(s.T(), err)
 
-	logger.Infof("Done with automatic upgrade from %s to master", installer.Version1_9)
-	newFile := "post-upgrade-1_9-to-master-file"
+	logger.Infof("Done with automatic upgrade from %s to master", installer.Version1_10)
+	newFile := "post-upgrade-previous-to-master-file"
```

Comment:
one comment, here we are using `previous` instead of `1_20` for consistency?

Reply:
My goal is to change as little as possible when updating the upgrade test for each release. This filename really doesn't need the version name in it, so I made it more generic.